azkaban-common/src/main/java/azkaban/executor/container/KubernetesContainerizedImpl.java

```diff
@@ -1089,12 +1098,13 @@ private void deletePod(final int executionId) throws ExecutorManagerException {
   /**
    * This method is used to delete service in Kubernetes which is created for Pod.
    *
-   * @param executionId
+   * @param flow
    * @throws ExecutorManagerException
    */
-  public void deleteService(final int executionId) throws ExecutorManagerException {
-    final String serviceName = getServiceName(executionId);
```

Comment:
Nit: Any reason for moving this inside the try block?

Reply:
A mistake :)